Gemfile

```diff
@@ -3,14 +3,14 @@ source 'http://rubygems.org'
 ruby '2.2.3'
 
 group :development do
-  gem 'activesupport'
-  gem 'cucumber'
-  gem 'cucumber_lint'
-  gem 'kappamaki'
-  gem 'mortadella'
-  gem 'open4'
-  gem 'parallel_tests'
-  gem 'rake'
-  gem 'rubocop'
-  gem 'rspec'
+  gem 'activesupport', '4.2.6'
```

Comment:
The main reason I did this was it was actually able to control which versions of which get installed via `bundle install`. When I ran bundle update, it updated cucumber and rolled back cucumber_lint in order for the two to use a compatible gherkin. It doesn't necessarily need to be locked down here but following what we did with node, why not also lock down here. That way you don't need to look at the gemfile.lock to see what versions of what you are using.